packages/replay-data/src/recordingData/PointQueries.ts

```diff
@@ -58,6 +58,13 @@ const debug = createDebug("replay:PointQueries");
 
 const POINT_ANNOTATION = "/*POINT*/";
 
+export function omitPoint(reason?: string): string {
```

Comment:
NOTE: This can be very useful to prevent the agent from getting stuck in third-party code. But it can cause problems where a (generally shallow) investigation of third-party code is actually necessary. We'll have to keep playing with this as we move along.